lessons/MachineLearning/lesson.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Machine Learning
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# My Lesson Title
+
+ - **Authors**: Nil Sahin
+ - **Research field**: Computational Biology and Molecular Genetics
+ - **Lesson topic**: Machine Learning using Python's Sklearn packages
+ - **Lesson content URL**: <link>
```

Comment:
please add the link: `<https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/misc/scikit-learn>`